src/styles/main.scss

```diff
@@ -1,3 +1,37 @@
-body {
+$block-size: 300px;
+
+* {
+  box-sizing: border-box;
+}
+
+.page-root {
   margin: 0;
+  font-family: Arial, sans-serif;
+  font-size: 100px;
```

Comment:
add this in html tag, and so everything else is fine)

Reply:
fixed